modules/dnds/1.0/config/default.yaml

```diff
@@ -0,0 +1,42 @@
+lcr-modules:
+
+    dnds:
+
+        inputs:
+            # Available wildcards: {seq_type}
+            master_maf: "__UPDATE__"
+            sample_sets: "__UPDATE__"
+        sample_set: ["__UPDATE__"]
+        seq_types: ["genome", "capture"]
+        include_non_coding: True
+        prepare_mafs: "{SCRIPTSDIR}/generate_smg_inputs/1.0/generate_smg_inputs.R"
+        src_dir: "{MODSDIR}/src/"
+
+        options:
+            target_genes: "" # optionally, specify path to the genes of interest if this is a targeted sequencing
+                            # this will restrict SMG search to only these genes
+                            # this is a tsv file that contain one gene symbol/row and is expected to be without header
+            max_muts_per_gene_per_sample: Inf # How many mutations/gene dNdS should consider?
+                                            # This default value will consider all mutations/gene.
+                                            # When modified, this should be integer number.
+
+        conda_envs:
+            prepare_mafs: "{REPODIR}/envs/gatk/gatkR.yaml"
```

Comment:
is REPODIR a new variable now?

Reply:
Not exactly, it is just not used very often. This is a shortcut to the parent directory of lcr-modules. More details here: https://lcr-modules.readthedocs.io/en/latest/for_users.html?highlight=repodir#directory-placeholders